cinema/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True)
+    description = serializers.TextField(required=True)
+    duration = serializers.IntegerField(required=False)

```

Comment:
duration is required, description is not

Reply:
Fixed